README.md

```diff
@@ -17,6 +17,17 @@ Or using place search:
 $ set-simulator-location -q Lyft HQ San Francisco
 ```
 
+By default the location is set on all booted simulators. If you'd like
+to change it for only one of the booted simulators you can pass `-s`
+followed by the simulator's display name:
+
+```sh
+$ set-simulator-location -q Lyft HQ San Francisco -s iPhone X
+```
+
+NOTE: If you have multiple booted simulators with the same name, the
```

Comment:
I'm surprised they're not required to be unique. Are there known use cases that benefit from duplicated names?

Reply:
Xcode de-dups them in the UI by showing the UDID as well. But if you don't add custom names this is the default when you add another device of the same time